src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard {...movie} key="movie.imdbId" />)}
```

Comment:
Здається, форматування тут не найкраще)

Reply:
it's fine

Reply:
@aliceblunt8 Хіба от так не треба?
```suggestion
    {
      movies.map(movie => <MovieCard {...movie} key="movie.imdbId" />)
    }
```

Reply:
@aliceblunt8 Хіба от так не треба?
```suggestion
    {
      movies.map(movie => <MovieCard {...movie} key="movie.imdbId" />)
    }
```

Reply:
@putsan це не обо'язково